app/controllers/application_controller.rb

```diff
@@ -11,6 +11,15 @@ class ApplicationController < ActionController::Base
     redirect_to root_url, alert: exception.message
   end
 
+  def after_sign_in_path_for(user)
```

Comment:
Can you add a spec for this please?

The spec should exercise the basic behaviour and also check the existing redirects we do. See HomeController.redirect_if_logged_in.

E.g. spec should check that a) user gets directs to Welsh site and b) then gets redirected to school home page or whatever destination is correct.

Reply:
Also, thinking we only need to do this redirect if they user isn't already on the right site?

Reply:
:+1: some tests added.  i think we're ok here for the redirect will resolve to the same path.